cackle.toml

```diff
@@ -266,6 +266,7 @@ allow_unsafe = true
 build.allow_apis = [
     "env",
     "process",
+    "terminate",
```

Comment:
This is a bit concerning, how come it appeared in the *older* version of the library and we didn't need it before?

Reply:
I didn't look into how this is being used and not sure how concerning this is. But I could roll back to the previous version that doesn't require it.